bundles/org.openhab.binding.broadlinkthermostat/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+	https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:broadlinkthermostat:floureonandhysenthermostat">
+		<parameter name="host" type="text">
+			<label>Hostname</label>
+			<description>The hostname/IP address the device is bound to, e.g. 192.168.0.2</description>
+			<required>false</required>
```

Comment:
The `required` tag is deprecated. You could use the attribute instead: `required="true"`

Reply:
changed to the new attribute.